docs-site/docs/dist-train.md

```diff
@@ -75,12 +75,30 @@ sgd = opt.DistOpt(sgd)
 dev = device.create_cuda_gpu_on(sgd.rank_in_local)
 ```
 
-`dev` represents the `Device` instance, where to load data and run the CNN
+Here are some explanations concerning some variables in the code:
+
+(i) `dev` 
+
+dev represents the `Device` instance, where to load data and run the CNN
 model.
 
+(ii)`rank_in_local` 
```

Comment:
local_rank
global_rank

Reply:
yes, we use <local_rank, global_rank, world size> everywhere